README.md

```diff
@@ -256,6 +257,32 @@ function MDXPage({code}: {code: string}) {
 }
 ```
 
+### Component Substitution
+
+MDX Bundler passes on
+[XDM's ability to substitute components](https://github.com/wooorm/xdm#mdx-content)
+through the `components` prop on the component returned by `getMDXComponent`.
+
+Here's an example replacing `<b>` with `<strong>`:
+
+```tsx
+import * as React from 'react'
+
+const Strong: React.FC = props => {
+  return <strong {...props} />
+}
+
+function MDXPage({code}: {code: string}) {
+  const Component = React.useMemo(() => getMDXComponent(code), [code])
+
+  return (
+    <main>
+      <Component components={{b: Strong}} />
```

Comment:
Could you check whether this works as well?

```ts
<Component components={{b: 'strong'}} />
```

I have a feeling the way things are implemented that would work too, in which case our typings should allow this. Also, maybe we should come up with a less contrived example of this feature so people get an idea of why you might want to use this.

Reply:
`<Component components={{b: 'strong'}} />` does work, I've added `string` to the types.

I've swapped out the example for one from my site which removes the _p_ tags from around images.